frontend/src/app.config.ts

```diff
@@ -58,5 +59,6 @@ config["eventListenerJobStatus"]["skipped"] = "SKIPPED";
 config["eventListenerJobStatus"]["resubmitted"] = "RESUBMITTED";
 
 config["streamingBytes"] = 1024 * 10; // 10 MB?
+config["rawDataVisualizationThreshold"] = 1024 * 1024 * 10; // 10 MB
```

Comment:
You might be able to just use the "streamingBytes" ^. I remembered i might reserved that but never used it

Reply:
I didn't realize you don't use streamingBytes.